sample-code/javascript-wd/test/basic/android-create-session.test.js

```diff
@@ -0,0 +1,30 @@
+import wd from 'wd';
+import chai from 'chai';
+import { iosCaps, androidCaps, serverConfig } from '../helpers/caps';
+
+const {assert} = chai;
+
+describe('Create Android session', function () {
+  it('should create and destroy IOS sessions', async function () {
+    // Connect to Appium server
+    const driver = await wd.promiseChainRemote(serverConfig);
+
+    // We haven't started a session yet, so we shouldn't see any sessions running
+    assert.equal((await driver.sessions()).length, 0);
+
+    // Start the session
+    await driver.init({
+      ...androidCaps,
+      app: require('../helpers/apps').androidApiDemos
+    });
+
```

Comment:
Might be useful to check that the App running is the app we expect (e.g., on Android, get the running package and activity; on iOS just check that the `XCUIElementTypeApplication` element is what we expect).